ParraCore/Types/Configuration/ParraConfiguration.swift

```diff
@@ -11,12 +11,14 @@ import Foundation
 public struct ParraConfiguration {
     public let loggerConfig: ParraLoggerConfig
     public internal(set) var tenantId: String?
+    public internal(set) var applicationId: String?
 
     // Public version of this initializer should be kept up to date to include
-    // all fields except for tenantId.
+    // all fields except for tenantId and applicationId
     public init(loggerConfig: ParraLoggerConfig) {
         self.loggerConfig = loggerConfig
         self.tenantId = nil
+        self.applicationId = nil
```

Comment:
Could this and tenantId not be included in this initializer?

Reply:
@ianmaccallum They, _could_ be but I think this makes more sense with how we're doing initialization at the moment. tenant/application id are both parameters of the auth provider since they're tied to auth, the the config object is just a way of moving them around/letting an end user see what they're currently set to. Since this is the public facing initializer we don't want them to be overridden here.